mothulity/mothulity

```diff
@@ -334,6 +334,22 @@ def main():
         Default <~/db/Silva.nr_v119/silva.nr_v119.tax>.
         """
     )
+    mothur.add_argument(
+        "--alignment-region",
+        action="store",
+        dest="alignment_region",
+        metavar="6388-13861",
```

Comment:
```metavar``` is displayed next to the option name. For the user, it looks just like a default value and is misleading. Please set it to empty string.

Reply:
You're right, it looks really messy. Fixed.